### DISCUSSION TITLE
Find only those parents records who has atleast one child recored

### DISCUSSION THREAD

**Discussion by ShahrukhMobilecoderz** [adonisjs/core#1451] [Help] [answered]
I have reviews table and review_images table. Review is a parent table and review_images is a child table. I need to find only those reviews who has atleast one review images using one is to many relationship. Kindly help me on this.

**Comment by thetutlage** [comment, marked as answer]
Which version of the framework?

**Reply by ShahrukhMobilecoderz** [reply]
Adonisjs 4.1.

**Reply by thetutlage** [reply]
Covered here in the docs https://adonisjs.com/docs/4.0/relationships#_has

**Reply by ShahrukhMobilecoderz** [reply]
Thanks. It is working. I have used whereHas to achieve this.
